extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -530,14 +579,9 @@ The information is available either in `state.data.details` from the `onAddition
 }
 ```
 
-Extension module will extend `submitAdditionalPaymentDetailsRequest` with `paymentData` attribute if the attribute is missing.
-In this case, `paymentData` will be taken from the previous `makePaymentRequest`.
-
-After update, you will receive `submitAdditionalPaymentDetailsResponse` in the returned commercetools payment.
-The next steps depend on the existence of an action object within `submitAdditionalPaymentDetailsResponse`.
-
-If you received an action object, [pass the action object to your front end](https://docs.adyen.com/checkout/components-web/#step-4-additional-front-end) and perform Step 4 again.
-Submit additional payment details response from Adyen for the case where you need to pass the action object to your front end:
+Extension module will extend `submitAdditionalPaymentDetailsRequest` with `paymentData` attribute if the attribute is missing. In this case, `paymentData` will be taken from the previous `makePaymentRequest`.
+After update, you will receive `submitAdditionalPaymentDetailsResponse` in the returned commercetools payment. The next steps depend on the existence of an action object within `submitAdditionalPaymentDetailsResponse`.
+If you received an action object, [pass the action object to your front end](https://docs.adyen.com/checkout/components-web/#step-4-additional-front-end) and perform Step 4 again. Submit additional payment details response from Adyen for the case where you need to pass the action object to your front end:
```

Comment:
deep link does not work.

Reply:
`Submit additional payment details response from Adyen for the case where you need to pass the action object to your front end`

Not sure what needs to be done here.